src/api/TokenIO.h

```diff
@@ -170,6 +170,39 @@ globalRpcErrorCallback:(OnError)globalRpcErrorCallback;
                           onSuccess:(OnSuccess)onSuccess
                             onError:(OnError)onError;
 
+#pragma mark - Recovery
+
+/**
+ * Begins recovery process for an alias. The verification message will be sent if the alias is valid.
+ *
+ * @param aliasValue alias value to recover
+ * @param onSuccess invoked if successful
+ * @param onError invoked if failed
+ */
+- (void)beginRecovery:(NSString *)aliasValue
+            onSuccess:(OnSuccess)onSuccess
+              onError:(OnError)onError;
+
+/**
+ * Verifies recovery code after beginRecovery is successful.
+ *
+ * @param code code from verification message
+ * @param onSuccess invoked if successful
+ * @param onError invoked if failed
+ */
+- (void)verifyRecoveryCode:(NSString *)code
+                 onSuccess:(OnSuccessWithBoolean)onSuccess
+                   onError:(OnError)onError;
+
+/**
+ * Completes recovery process after verifyRecoveryCode is successful.
+ *
+ * @param onSuccess invoked if successful with TkMember
+ * @param onError invoked if failed
+ */
+- (void)completeRecovery:(OnSuccessWithTKMember)onSuccess
```

Comment:
If the recovery manager is not exposed to the developer, then we might lose the recovery payload between the verify and complete steps. It is also unclear that these calls need to be executed with the same instance of TokenIO. Should we have a method in TokenIO that returns a RecoveryManager that the developer can then use to complete the recovery process?

Reply:
saving an TokenIO instance is nothing different with saving an RecoveryManager. I would not suggest the developer to learn another things called recovery manager. 
To solve this, I will add comments in recovery methods that developers shall use the same tokenio instance.